themes/components-training-theme/src/css/components/_type.scss

```diff
@@ -0,0 +1,132 @@
+h1, .h1, h1#heading1 { 
```

Comment:
It's missing the fonts files and @font-face for each Source Sans Pro font

references: 

- https://fonts.google.com/specimen/Source+Sans+Pro
- https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face